packages/light-apps/src/Content/Content.tsx

```diff
@@ -42,6 +42,7 @@ export class Content extends React.Component<Props> {
               <Route component={IdentityCard} />
               <Switch>
                 <Redirect exact from='/' to={`/identity/${defaultAddress}`} />
+                <Redirect exact from='/' to={`/transfer/${defaultAddress}`} />
```

Comment:
This one will never be called. The first one says "redirect / to /identity/...", you cannot redirect / to 2 different locations